packages/web3-eth/test/integration/unsubscribe.test.ts

```diff
@@ -2,17 +2,22 @@ import WebSocketProvider from 'web3-providers-ws/dist';
 import { SupportedProviders } from 'web3-core';
 import Web3Eth from '../../src/index';
 import { NewHeadsSubscription, SyncingSubscription } from '../../src/web3_subscriptions';
+// eslint-disable-next-line
```

Comment:
You should disable the specific rule that's being violated, not completely disable the linter